CentralManagement/ChocoCCM Command Reference/Remove-CCMDeployment.md

```diff
@@ -0,0 +1,91 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Remove-CCMDeployment
+
+## SYNOPSIS
+Removes a deployment plan
+
+## SYNTAX
+
+```
+Remove-CCMDeployment [-Deployment] <String[]> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Removes the Deployment Plan selected from a Central Management installation

```

Comment:
I don't think you can remove deployments that have already run, right? It would be good to mention that in the info here and inform folks of the archive command or something.

(Also, "deployment plan" should probably be just "deployment" here as well.)